<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Codys-Mac.local" ignored="0" name="Command line test 08704cf4-fc9d-42c3-83c0-24f0a8e44467" tests="2" failures="2" timestamp="27 Mar 2018 08:02:16 GMT" time="524.241" errors="0">
  <testcase name="prankcaller" time="273.235" classname="test.tests.launchApp.Tests">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at test.tests.launchApp.Tests.prankcaller(Tests.java:230)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- prankcaller -->
  <testcase name="launchApp" time="101.02" classname="test.tests.launchApp.Tests">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at test.tests.launchApp.Tests.launchApp(Tests.java:421)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- launchApp -->
</testsuite> <!-- Command line test 08704cf4-fc9d-42c3-83c0-24f0a8e44467 -->
